cc/toolchains/musl/armhf/config.bzl

```diff
@@ -0,0 +1,139 @@
+load("@bazel_tools//tools/build_defs/cc:action_names.bzl", "ACTION_NAMES")
+
+SDK_PATH_PREFIX = "wrappers/arm-linux-musleabihf-{}"
+
+def _impl(ctx):
+    tool_paths = [
+        tool_path(
+            name = "ar",
+            path = SDK_PATH_PREFIX.format("ar"),
+        ),
+        tool_path(
+            name = "as",
+            path = SDK_PATH_PREFIX.format("as"),
+        ),
+        tool_path(
+            name = "cpp",
+            path = SDK_PATH_PREFIX.format("cpp"),
+        ),
+        tool_path(
+            name = "gcc",
+            path = SDK_PATH_PREFIX.format("gcc"),
+        ),
+        tool_path(
+            name = "g++",
+            path = SDK_PATH_PREFIX.format("g++"),
+        ),
+        tool_path(
+            name = "gcov",
+            path = SDK_PATH_PREFIX.format("gcov"),
+        ),
+        tool_path(
+            name = "ld",
+            path = SDK_PATH_PREFIX.format("ld"),
+        ),
+        tool_path(
+            name = "nm",
+            path = SDK_PATH_PREFIX.format("nm"),
+        ),
+        tool_path(
+            name = "objcopy",
+            path = SDK_PATH_PREFIX.format("objcopy"),
+        ),
+        tool_path(
+            name = "objdump",
+            path = SDK_PATH_PREFIX.format("objdump"),
+        ),
+        tool_path(
+            name = "ranlib",
+            path = SDK_PATH_PREFIX.format("ranlib"),
+        ),
+        tool_path(
+            name = "strip",
+            path = SDK_PATH_PREFIX.format("strip"),
+        ),
+    ]
+
+    features = [
+        feature(
+            name = "default_compile_actions",
+            enabled = True,
+            flag_sets = [
+                flag_set(
+                    actions = [
+                        ACTION_NAMES.assemble,
+                        ACTION_NAMES.c_compile,
+                        ACTION_NAMES.cpp_compile,
+                        ACTION_NAMES.cpp_header_parsing,
+                        ACTION_NAMES.cpp_module_codegen,
+                        ACTION_NAMES.cpp_module_compile,
+                        ACTION_NAMES.clif_match,
+                        ACTION_NAMES.linkstamp_compile,
+                        ACTION_NAMES.lto_backend,
+                        ACTION_NAMES.preprocess_assemble,
+                    ],
+                    flag_groups = ([
+                        flag_group(
+                            flags = [
+                                "--sysroot=external/arm-linux-musleabihf",
+                                "-no-canonical-prefixes",
+                                "-fno-canonical-system-headers",
+                                "-fno-common",
+                                "-ffunction-sections",
+                                "-fdata-sections",
+                                # Reproducibility
+                                "-Wno-builtin-macro-redefined",
+                                "-D__DATE__=\"redacted\"",
```

Comment:
Does this affect tests that use the time to the fail the build, like our leap second check?